engine/class_modules/sc_hunter.cpp

```diff
@@ -6441,60 +6441,62 @@ void hunter_t::apl_surv()
   action_priority_list_t* moknathal    = get_action_priority_list( "moknathal" );
   action_priority_list_t* nomok        = get_action_priority_list( "nomok" );
 
-  precombat -> add_action( "explosive_trap,if=!talent.steel_trap.enabled" );
+  precombat -> add_action( "explosive_trap" );
   precombat -> add_action( "steel_trap" );
   precombat -> add_action( "dragonsfire_grenade" );
   precombat -> add_action( "harpoon" );
 
   default_list -> add_action( "auto_attack" );
 
-  add_racial_actions( default_list );
```

Comment:
no, both blood fury & berserking are added a couple lines below
if we leave this here the newly added ones will basically have no effect as the "generic" ones are less strict